config/runtime/tests/fluster-chromeos.jinja2

```diff
@@ -45,17 +45,26 @@
               root@$(lava-target-ip)
               python3 /usr/bin/fluster_parser.py {{ testsuite_arg }} {{ decoders_arg }} 
                             {{ videodec_timeout_arg }} {{ videodec_parallel_jobs_arg }}
+            - lava-test-set start get_results
+            - mkdir -p /opt/fluster/
             - >-
-              ./ssh_retry.sh
+              scp
               -o StrictHostKeyChecking=no
               -o UserKnownHostsFile=/dev/null
               -i /home/cros/.ssh/id_rsa
-              root@$(lava-target-ip)
-              python3 /usr/bin/fluster_parser.py --results
+              root@$(lava-target-ip):/opt/fluster/results.xml /opt/fluster/
+            - >-
+              scp
```

Comment:
Would it be useful to ship this tool in the TestJob environment by default, i.e. add it to the `kernelci/cros-tast` Docker container image?

Definitely out of scope for this PR but @denisyuji please share your thoughts